master/buildbot/steps/trigger.py

```diff
@@ -63,7 +65,17 @@ def __init__(self, schedulerNames=None, sourceStamp=None, sourceStamps=None,
             config.error(
                 "You can't specify both alwaysUseLatest and updateSourceStamp"
             )
+        if len(set(schedulerNames) & set(unimportantSchedulerNames)) != len(unimportantSchedulerNames):
+            config.error(
+                "unimportantSchedulerNames must be a subset of schedulerNames"
+            )
+        if len(schedulerNames) <= len(unimportantSchedulerNames):
+            config.error(
+                    "SchedulerNames must at least contain an important scheduler"
+            )
+
         self.schedulerNames = schedulerNames
+        self.unimportantSchedulerNames = unimportantSchedulerNames
```

Comment:
should we store a set instead? (it's already used above and then it'd be handy in the code below)